CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ All notable, unreleased changes to this project will be documented in this file.
 # 3.16.0 [Unreleased]
 
 ### Breaking changes
+- Change the Attribute - Product relation to decrease code complexity and make it easier to understand the relations - #13407 by @aniav
+- The CSV export will now use empty string for empty values instead of a whitespace value
```

Comment:
Is it an expected changelog here?

Reply:
I see that you also touched the csv logic, and it looks like this changelog is valid. I am not sure that we should place such changes in the PR related to Attributes. I would never think that Attribute refactors could include the changes in CSV.
Also, this is a breaking change, that could have an impact on existing services that consume our CSV, is that correct? In that case, we should think about how we provide backward compatibility, I don't think that we can just change it. I know that this is a small change but still, it can be a breaking change.

Reply:
While working on the changes in my PR I noticed we have two types of empty values in the CSV exports: whitespace for situations where the value was not assigned and an empty string for situations where the value is assigned and is effectively empty. If I wanted to keep this state it would require a lot of unwanted changes in my PR so I decided to also incorporate it here.

I agree it doesn't belong here. I'll create a separate PR for these changes to be reviewed before going back to this PR.

Reply:
Moved to: https://github.com/saleor/saleor/pull/13779